pkg/internal/apis/namevaluepair.go

```diff
@@ -13,11 +13,14 @@ limitations under the License.
 
 package apis
 
-import "github.com/dapr/dapr/pkg/apis/common"
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/dapr/dapr/pkg/apis/common"
+)
 
 type GenericNameValueResource struct {
-	Name         string
-	Namespace    string
+	metav1.ObjectMeta
```

Comment:
Why is this needed?

Reply:
Reverted to only have the `Name` and `Namespace` fields.